### PR TITLE
Updated default plans 

### PR DESCRIPTION
**Pull request by Bertie690**
Ported over the base battle plans from vanilla stars & improved the default production plan to only terraforming tasks if neither CA nor tri-immune (maybe a bit redundant, but it never hurts to have extra checks). 
Also renamed the default empty transport plan to "Clear All". 
Fixes issue #318

### CODE REVIEW

File: cs/player.go [hunk 1]
```diff
@@ -589,23 +589,34 @@ func (p *Player) defaultRelationships(players []*Player, aiFormsAlliances bool)
...
 
 	// AR races don't build factories or mines
+	// CA & tri-immune races don't do terraforming
 	defaultProductionPlan := ProductionPlan{
 		Num:  0,
 		Name: "Default",
-		Items: []ProductionPlanItem{
-			{Type: QueueItemTypeAutoMinTerraform, Quantity: 1},
-		},
+		Items: []
```
Comment (sirgwain) [inline cs/player.go]: needs a comma

**Review by sirgwain — CHANGES_REQUESTED** [review]
looks good, but can you add the missing commas so it compiles?

File: cs/player.go (hunk 1 above)
Comment (Bertie690) [reply cs/player.go]: Done

**Comment by Bertie690** [comment]
Oke dokey fixed it

File: cs/player.go [hunk 2]
```diff
@@ -589,23 +589,34 @@ func (p *Player) defaultRelationships(players []*Player, aiFormsAlliances bool)
...
 
 	// AR races don't build factories or mines
+	// CA & tri-immune races don't do terraforming
 	defaultProductionPlan := ProductionPlan{
 		Num:  0,
 		Name: "Default",
-		Items: []ProductionPlanItem{
-			{Type: QueueItemTypeAutoMinTerraform, Quantity: 1},
-		},
+		Items: [],
```
Comment (sirgwain) [inline cs/player.go]:
```suggestion
		Items: []ProductionPlanItem{},
```

File: cs/player.go [hunk 3]
```diff
@@ -621,11 +632,43 @@ func (p *Player) defaultPlans() PlayerPlans {
...
+				Tactic:		 BattleTacticMaximizeNetDamage,
+				AttackWho:	 BattleAttackWhoEnemiesAndNeutrals,
+			},
+			{
+				Num:		 3,
+				Name: 		 "Sniper",
+				PrimaryTarget:	 BattleTargetUnarmedShips,
+				SecondaryTarget: BattleTargetNone,
+				Tactic:		 BattleTacticDisengageIfChallenged,
+				AttackWho:	 BattleAttackWhoEnemiesAndNeutrals
```
Comment (sirgwain) [inline cs/player.go]:
```suggestion
				AttackWho:	 BattleAttackWhoEnemiesAndNeutrals,
```

File: cs/player.go [hunk 4]
```diff
@@ -669,7 +712,7 @@ func (p *Player) defaultPlans() PlayerPlans {
 				Num:  4,
 				Name: "Unload Colonists",
 				Tasks: WaypointTransportTasks{
-					Colonists: WaypointTransportTask{Action: TransportActionUnloadAll},
+					Colonists: WaypointTransportTask{Action: TransportActionLoadDunnage},
```
Comment (sirgwain) [inline cs/player.go]: Why load dunnage instead of unload all?

**Review by sirgwain — CHANGES_REQUESTED** [review]

File: cs/player.go (hunk 4 above)
Comment (Bertie690) [reply cs/player.go]: I guess I typo'd a bit...
Will fix

**Review by sirgwain — APPROVED** [review]

**Pull request merged**